cost/plan.go

```diff
@@ -52,10 +52,39 @@ func (p Plan) ResourceDifferences() []ResourceDiff {
 	return rds
 }
 
+// SkippedAddresses returns the addresses of resources that were excluded from the estimation process.
+// The order of the elements in the slice is undefined and unstable.
+func (p Plan) SkippedAddresses() []string {
+	skippedMap := make(map[string]struct{})
+	if p.Prior != nil {
+		for addr, res := range p.Prior.Resources {
+			if res.Skipped {
+				skippedMap[addr] = struct{}{}
+			}
+		}
+	}
+	if p.Planned != nil {
+		for addr, res := range p.Planned.Resources {
+			if res.Skipped {
+				skippedMap[addr] = struct{}{}
+			}
+		}
+	}
+	skippedList := make([]string, 0, len(skippedMap))
+	for addr := range skippedMap {
+		skippedList = append(skippedList, addr)
+	}
+	return skippedList
```

Comment:
Maybe the order is something we want to have static, so a `sort.Strings` may be good :)